inspec.yml

```diff
@@ -0,0 +1,41 @@
+name: cms-ars-3.1-moderate-aws-rds-microsoft-sql-server-2017-cis-overlay
+title: cms-ars-3.1-moderate-aws-rds-microsoft-sql-server-2017-cis-overlay
+maintainer: MITRE SAF
+copyright: MITRE SAF
+copyright_email: saf@groups.mitre.org
+license: Apache-2.0
+summary: "An InSpec Overlay Profile for CMS ARS 3.1 Moderate to validate the Microsoft SQL Server 2017 CIS Benchmark"
+version: 1.2.0
+inspec_version: ">= 4.0"
+
+depends:
+- name: microsoft-sql-server-2017-cis-baseline
+  url: https://github.com/mitre/microsoft-sql-server-2017-cis-baseline/archive/v1.2.0.tar.gz
```

Comment:
I'll update the branch name in the `depends.url` when the PR for the baseline has been approved and merged. Thanks!

Reply:
Branch name updated!